testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksInsertCompatibility.java

```diff
@@ -411,7 +413,7 @@ private void testCompression(boolean optimizedWriter, String compressionCodec)
                 assertThat(onTrino().executeQuery("SELECT * FROM " + trinoTableName))
                         .containsOnly(expected);
 
-                if ("ZSTD".equals(compressionCodec) && !databricksRuntimeVersion.equals(DATABRICKS_104_RUNTIME_VERSION)) {
+                if ("ZSTD".equals(compressionCodec) && databricksRuntimeVersion.orElseThrow().isOlderThan(DATABRICKS_104_RUNTIME_VERSION)) {
```

Comment:
if `databricksRuntimeVersion` must be present, can you make the field non-Optional, as you did in other tests?

Reply:
This class has `DELTA_LAKE_OSS` group, so we can't make it non-Optional in the field.